volume/drivers/pwx/connection.go

```diff
@@ -181,6 +181,14 @@ func (cpb *ConnectionParamsBuilder) BuildTlsConfig() (*tls.Config, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	if upd, err := cpb.appendK8sCaCertBytes(rootCA); err == nil {
+		logrus.Debugf("Found Kubernetes CA, adding into TLS config")
+		rootCA = upd
+	} else if !os.IsNotExist(err) {
+		logrus.WithError(err).Warnf("Could not append Kubernetes CA")
```

Comment:
if the user intended to setup a rootCA but we can't find it, shouldn't we return error here? Otherwise we'll silently fail and not setup properly

Reply:
The `cpb.appendK8sCaCertBytes()` appends the Kubernetes CA, and it's meant to be "opportunistic" --
* e.g. if file the is there (e.g. if we're running in Kubernetes env), go ahead and [always] _append_ K8s CA
  - but, if the file was not there (e.g. non-K8s env's), silently ignore the error
* note that it'll add this CA _in addition_ to CA-certs provided via secret
  - see `cpb.getCaCertBytes()` -- note that this func always returns its errors  (does not ignore them)